changeset/commit.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package changeset
```

Comment:
This is great, I think the one thing I'd like to see us add is a `doc.go` file with a package comment that has a detailed explanation of how to set this up (e.g. explaining the need for `ln -s -r .git/HEAD ./cmd/foo/kodata/`)

Reply:
Done.